portfolio/src/main/webapp/script.js

```diff
@@ -28,24 +28,26 @@ function addRandomGreeting() {
 }
 
 function displayHobbies() {
-    const title = document.getElementById('title');
+    const title = document.getElementById('content-title');
     title.innerText = "This is a list of my main hobbies:";
```

Comment:
Since both sections are very similar, you could actually try to "extract" a common function, like:
 `updateSection(filename, title) { ...} `

This will avoid repeating the DOM manipulation twice, you can just
```
function displayHobbies() {
  updateSection('main_page_clickables/hobbies.html', 'My hobbies....');
}
```